src/component/SearchResults.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import Moment from "react-moment";
+
+function CreateRowTable({ customer }) {
+  const [select, setselect] = useState(false);
```

Comment:
I'd think of a more descriptive name of the variable here to reflect the fact that it's a boolean. Something like `isSelected`. And also remember to use **camelCase** for the setter function.